data/schemas/schemas-microsoft-com_office_word.json

```diff
@@ -0,0 +1,365 @@
+{
+  "TargetNamespace": "urn:schemas-microsoft-com:office:word",
+  "Types": [
+    {
+      "Name": "w10:CT_Border/w10:bordertop",
+      "ClassName": "TopBorder",
+      "Summary": "Top Border.",
+      "BaseClass": "BorderType",
+      "IsDerived": true,
+      "IsLeafElement": true
+    },
+    {
+      "Name": "w10:CT_Border/w10:borderleft",
+      "ClassName": "LeftBorder",
+      "Summary": "Left Border.",
+      "BaseClass": "BorderType",
+      "IsDerived": true,
+      "IsLeafElement": true
+    },
+    {
+      "Name": "w10:CT_Border/w10:borderright",
+      "ClassName": "RightBorder",
+      "Summary": "Right Border.",
+      "BaseClass": "BorderType",
+      "IsDerived": true,
+      "IsLeafElement": true
+    },
+    {
+      "Name": "w10:CT_Border/w10:borderbottom",
+      "ClassName": "BottomBorder",
+      "Summary": "Bottom Border.",
+      "BaseClass": "BorderType",
+      "IsDerived": true,
+      "IsLeafElement": true
+    },
+    {
+      "Name": "w10:CT_Border/",
+      "ClassName": "BorderType",
+      "Summary": "Defines the BorderType Class.",
+      "BaseClass": "OpenXmlLeafElement",
+      "IsAbstract": true,
+      "IsLeafElement": true,
+      "Attributes": [
+        {
+          "QName": ":type",
+          "PropertyName": "Type",
+          "Type": "EnumValue\u003CDocumentFormat.OpenXml.Vml.Wordprocessing.BorderValues\u003E",
+          "PropertyComments": "Border Style"
+        },
+        {
+          "QName": ":width",
+          "PropertyName": "Width",
+          "Type": "IntegerValue",
+          "PropertyComments": "Border Width",
+          "Validators": [
+            {
+              "Arguments": [
+                {
+                  "Type": "Boolean",
+                  "Name": "IsPositive",
+                  "Value": "True"
+                }
+              ],
+              "Name": "NumberValidator",
+              "IsInitialVersion": true
+            }
+          ]
+        },
+        {
+          "QName": ":shadow",
+          "PropertyName": "Shadow",
+          "Type": "TrueFalseValue",
+          "PropertyComments": "Border shadow"
+        }
+      ]
+    },
+    {
+      "Name": "w10:CT_Wrap/w10:wrap",
+      "ClassName": "TextWrap",
+      "Summary": "Text Wrapping.",
+      "BaseClass": "OpenXmlLeafElement",
+      "IsLeafElement": true,
+      "Attributes": [
+        {
+          "QName": ":type",
+          "PropertyName": "Type",
+          "Type": "EnumValue\u003CDocumentFormat.OpenXml.Vml.Wordprocessing.WrapValues\u003E",
```

Comment:
hm... again with the unicode chars. @twsouthwick do you know why this is?

Reply:
JSON encoding

Reply:
Yep, as above, both `Newtonsoft.Json` and `System.Text.Json` will deserialize as:

- `"EnumValue<DocumentFormat.OpenXml.Vml.Wordprocessing.WrapValues>"`

And `Newtonsoft.Json` would also not encode the `<` and `>` characters. 

The fact that `System.Text.Json` encodes many characters that are not encoded by `Newtonsoft.Json` leads to surprises for developers who are used to the `Newtonsoft.Json` behavior.

Reply:
Thanks Thomas and Taylor for the explanations.